### DISCUSSION TITLE
BlueSCSI v2 damaged

### DISCUSSION THREAD

**Discussion by David666d** [BlueSCSI/BlueSCSI-v2#201] [Help]
Hi there.
By mistake BlueSCSI v2 was connected opposite way (50 pin connector).
I damaged U16 - there was little crack on it and short on some capacitors. I ordered and replaced with similar chip. No short but no success. 
White diode light on - still. Blue flash. Green on PICO flash.
LXH125/43KG4/AL50 replaced with LHX125/25K/CL9R 
Any ideas? 
Could it be wrong chip that works wit different milliamperes? 
Is there any schematics for it?
Cheers 
David

Platform: BlueSCSI
FW Version: 2024.09.15-rel Sep 15 2024 14:49:19
I2C Supported
Flash chip size: 2048 kB
 
=== SD Card Info ===
SD card detected, FAT32 volume size: 29838 MB
SD Name: MSSD0, MID: 0xAD, OID: 0x4C 0x53
 
=== Global Config ===
Reading configuration from bluescsi.ini
Active configuration:
-- SelectionDelay: 0
-- EnableSelLatch is on
 
=== Finding images in / ===
== Opening /HD0-6.0.8-500M.hda for ID: 0 LUN: 0
---- Image ready

=== ROM Drive ===
Platform supports ROM drive up to 1692 kB
---- ROM drive image not detected
 
=== Configured SCSI Devices ===
* ID: 0, BlockSize: 512, Type: Fixed, Quirks: Apple, Size: 512000kB
 
Initialization complete!

**Comment by erichelgeson** [comment]
50 pin SCSI is built to survive an accidental reverse plugin of the connector, one side is signals, the other side is grounds or n/c. The term power pin is not connected on the flipped side. So I'm a bit perplexed as to what may have happened. 

Sounds like you're in contact with @flamelily already. If you do find a resolution or need additional help please let us know.

**Reply by erichelgeson** [reply]
Oh and schematics/design files/etc can be found here https://github.com/BlueSCSI/BlueSCSI-v2/tree/main/hardware for all models
